### ISSUE TITLE
Issue 404 response to GET /annotations/{id} where {id} doesn't exist

### ISSUE THREAD

**Issue by darrenleeweber** [sul-dlss/triannon#153]
The current response is a 200 OK when it should be a 404, e.g.
```
$ curl -I http://localhost:3000/annotations/xyz
HTTP/1.1 200 OK 
X-Frame-Options: SAMEORIGIN
X-Xss-Protection: 1; mode=block
X-Content-Type-Options: nosniff
Content-Type: application/ld+json; charset=utf-8
Etag: W/"9df5c3974b1caa6c7eebec7d20f4b8df"
Cache-Control: max-age=0, private, must-revalidate
X-Request-Id: 40a39171-6124-4478-87b9-0db756f22f95
X-Runtime: 0.041606
Server: WEBrick/1.3.1 (Ruby/2.1.5/2014-11-13)
Date: Thu, 02 Apr 2015 00:48:45 GMT
Content-Length: 0
Connection: Keep-Alive
Set-Cookie: request_method=HEAD; path=/
```

There is indeed a response body too, just a jsonld context, e.g.
```
$ curl http://localhost:3000/annotations/xyz
{"@context":"http://www.w3.org/ns/oa-context-20130208.json"}
```

**Comment by darrenleeweber** [comment]
The triannon console logs contain:
```
Started GET "/annotations/xyz" for 127.0.0.1 at 2015-04-01 17:50:03 -0700
Processing by Triannon::AnnotationsController#show as */*
  Parameters: {"id"=>"xyz"}
ERROR [line: 3] With input '<meta http-equiv="Content-Type" content="text/html; charset=ISO-8859-1"/>
<title>Error 404 Not Found<': Invalid token "<meta" (found "<meta"), production = :predicateObjectList
ERROR [line: 3] With input 'Error 404 Not Found</title>
</head>
<body><h2>HTTP ERROR 404</h2>
<p>Problem accessing /fedora/rest/a': Invalid token "Error" (found "Error"), production = :_triples_1
ERROR [line: 4] With input 'HTTP ERROR 404</h2>
<p>Problem accessing /fedora/rest/anno/xyz. Reason:
<pre>    Not Found</pre></p><': Invalid token "HTTP" (found "HTTP"), production = :objectList
ERROR [line: 5] With input 'Problem accessing /fedora/rest/anno/xyz. Reason:
<pre>    Not Found</pre></p><hr /><i><small>Powered ': Invalid token "Problem" (found "Problem"), production = :predicateObjectList
ERROR [line: 5] undefined prefix "Reason"
ERROR [line: 6] With input 'Not Found</pre></p><hr /><i><small>Powered by Jetty://</small></i><br/>                              ': Invalid token "Not" (found "Not"), production = :predicateObjectList
ERROR [line: 6] undefined prefix "Jetty"
ERROR [line: 6] With input '//</small></i><br/>                                                
<br/>                            ': Invalid token "//</small></i><br/>" (found "//</small></i><br/>"), production = :_triples_1
ERROR [line: 9] Unexpected (found "<br/>"(IRIREF)), production = "."
ERROR [line: 13] Unexpected (found "<br/>"(IRIREF)), production = "."
ERROR [line: 17] Unexpected (found "<br/>"(IRIREF)), production = "."
ERROR [line: 21] Unexpected (found "<br/>"(IRIREF)), production = "."
ERROR [line: 26] Unexpected (found "</body>"(IRIREF)), production = "."
ERROR [line: 27] Unexpected end of input, production = :predicateObjectList
Completed 200 OK in 26ms (Views: 0.1ms | ActiveRecord: 0.0ms)
```

**Comment by ndushay** [comment]
addressed with triannon gem version 0.7.2;  deployed to  dev  (and stage, which still isn't ready)
